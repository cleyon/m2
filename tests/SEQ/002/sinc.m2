@sequence s
We are at @++s@
Now we're @++s@
@incr s
@incr s
S is now @s@
