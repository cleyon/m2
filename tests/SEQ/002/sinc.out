We are at 1
Now we're 2
S is now 4
